GliaWidgets.xcodeproj/project.pbxproj

```diff
@@ -3944,7 +3947,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "make write-diff\n";
+			shellScript = "#make write-diff\n";
```

Comment:
Did you forget to uncomment this?

Reply:
Yes thank you!

Reply:
fixed [e84326b](https://github.com/salemove/ios-sdk-widgets/pull/705/commits/e84326b7cd81b80cecc5ec5b13b6881ff6057432)